TicTacToeUserInterface/UserInterface.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using TicTacToeApp;
+
+namespace TicTacToeUserInterface
+{
+    public class UserInterface
+    {
+
+        public TicTacToe newGame;
+
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+
+
+        public string startNewGame()
+        {
+            Console.WriteLine("Do you want to play a game of Tic Tac Toe? Y/N");
+            string answer = Console.ReadLine();
+
+            if (answer == "Y")
+            {
+                var marker = chooseMarker();
+                newGame = new TicTacToe(marker);
+                Console.WriteLine("A new game has been started");
+                return "A new game has been started";
+            }
+            else if (answer == "N")
+            {
+                return "Okay Bye!";
+            }
+            else
+            {
+                Console.WriteLine("Please enter Y or N only.");
+                return startNewGame();
+            }
+        }
+
+
+        public string chooseMarker()

```

Comment:
Good example for why a `Symbols` class would be nice. Notice in your code how many places you have "magic strings" where you check if something is equal to "X", or "O", or "_". If I said I wanted to use "+" and "o" for the symbols, you'd have to find every instance of those magic strings in your code which is tedious and error prone. If you instead replaced all those instances with `Symbols.X_MARKER` then you'd only have to make the change in one place. Though the name would be wrong then 🙂

Reply:
Cool! Yes I see the potential issues!